jcommune-view/jcommune-web-view/src/main/resources/org/jtalks/jcommune/web/view/messages_uk.properties

```diff
@@ -489,3 +489,8 @@ label.registration.success.2=
 label.not.logged.in.error=\u0421\u0445\u043E\u0436\u0435, \u0449\u043E \u0432\u0438 \u043D\u0435 \u043C\u0430\u0454\u0442\u0435 \u0431\u0456\u043B\u044C\u0448\u0435 \u043F\u0440\u0430\u0432 \u0437\u0430\u043B\u0438\u0448\u0430\u0442\u0438 \u043F\u043E\u0432\u0456\u0434\u043E\u043C\u043B\u0435\u043D\u043D\u044F \u0443 \u0446\u044C\u043E\u043C\u0443 \u0440\u043E\u0437\u0434\u0456\u043B\u0456. \u0417\u0431\u0435\u0440\u0435\u0436\u0456\u0442\u044C \u0442\u0435\u043A\u0441\u0442 \u0442\u0430 \u043F\u0435\u0440\u0435\u0437\u0430\u0432\u0430\u043D\u0442\u0430\u0436\u0442\u0435 \u0441\u0442\u043E\u0440\u0456\u043D\u043A\u0443 - \u043C\u043E\u0436\u043B\u0438\u0432\u043E \u0432\u0438 \u0431\u0456\u043B\u044C\u0448\u0435 \u043D\u0435 \u0430\u0432\u0442\u043E\u0440\u0438\u0437\u043E\u0432\u0430\u043D\u0456.
 label.search.user=\u041F\u043E\u0448\u0443\u043A \u043A\u043E\u0440\u0438\u0441\u0442\u0443\u0432\u0430\u0447\u0430
 label.search.user.empty=\u041A\u043E\u0440\u0438\u0441\u0442\u0443\u0432\u0430\u0447\u0456 \u043D\u0435 \u0437\u043D\u0430\u0439\u0434\u0435\u043D\u0456
+label.user.groups=\u0413\u0440\u0443\u043F\u043F\u044B
```

Comment:
Could you please ask to provide the translations on Russian, Ukrainian in the ticket?